tutorials/Threshold_Weighted_Scores.ipynb

```diff
@@ -5,14 +5,23 @@
    "metadata": {},
    "source": [
     "# Threshold weighted scores\n",
+    "\n",
     "## Introduction\n",
-    "For a long time, people have wanted to be able to evaluate the performance of forecasts of extreme events using verification methods that can't be hedged. Hedging occurs when there is a difference between a forecaster's judgement and their forecast [(Murphy 1978)](https://doi.org/10.1175/1520-0477(1978)059<0371:HATMOE>2.0.CO;2). In the context of an automated forecast system, hedging occurs when the forecast produced to optimise the expected verification score differs from a forecast produced that aligns with the prescribed forecast directive (e.g., forecast the 90th percentile wind speed). The threshold weighted scores in `scores` allow us to do this using consistent scoring functions.\n",
+    "For a long time, people have wanted to be able to evaluate the performance of forecasts of extreme events using verification methods that can't be hedged. This bring together two concepts - avoidance of hedging, particularly of probabilities, and also a focus on extreme conditions (e.g. very hot conditions).\n",
```

Comment:
"This bring together two concepts"

Perhaps we need to be more specific when we say "this". Perhaps "This tutorial" or "These threshold weighted scores",  or something else?

Reply:
By "this", I meant the evaluation of extreme events, but "this tutorial" and "these scores" would also be reasonable